File: backend/team6/src/test/java/programmers/team6/support/VacationTypeMother.java
package programmers.team6.support;

import org.jetbrains.annotations.NotNull;

import programmers.team6.domain.admin.entity.Code;

public class VacationTypeMother {

	public static Code Annual() {
		return new Code("VACATION_TYPE", "01", "연차");
	}

	@NotNull
	public static Code half() {
		return new Code("VACATION_TYPE", "05", "반차");
	}

	@NotNull
	public static Code reward() {
		return  new Code("VACATION_TYPE", "02", "포상 휴가");
	}
}

File: backend/team6/src/test/java/programmers/team6/support/VacationTypeMother.kt
package programmers.team6.support

import org.jetbrains.annotations.NotNull
import programmers.team6.domain.admin.entity.Code

object VacationTypeMother {
    fun Annual(): Code {
        return Code("VACATION_TYPE", "01", "연차")
    }

    @NotNull
    fun half(): Code {
        return Code("VACATION_TYPE", "05", "반차")
    }

    @NotNull
    fun reward(): Code {
        return Code("VACATION_TYPE", "02", "포상 휴가")
    }
}
